external-crates/move/crates/move-analyzer/prettier-plugin/src/printer.ts

```diff
@@ -129,7 +132,27 @@ export function print(path: AstPath, options: ParserOptions, print: printFn) {
                 ': ',
                 path.call(print, 'namedChildren', 1),
             ];
+        case 'positional_fields':
+            return breakable_comma_separated_list(path, node, '(', ')', print);
         default:
             return node.text;
     }
 }
+
+function breakable_comma_separated_list(path: AstPath,
+                                        node: SyntaxNode,
+                                        start: string,
+                                        end: string,
+                                        print: printFn) {
+
+    const items = Symbol('items');
+    return [
+        `${start}`,
```

Comment:
```suggestion
        start,
```

just that should work? same for `end`?

Reply:
Yeah, that was dumb :-) Initially I had a different implementation (of a different function that I subsequently re-wrote) where these included spaces before/after but at this point it just looks silly...